src/services/ivaDefectsService.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  DefectGETIVA,
+  EUVehicleCategory,
+  InspectionType,
+  VehicleType,
+} from "@dvsa/cvs-type-definitions/types/iva/defects/get/index";
+
+export class IvaDefectsService {
```

Comment:
Can you refactor this filename to IvaDefectService so it follows the same convention as DefectsService.